staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -152,6 +153,15 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 				options,
 			)
 		}
+		if accessor, accessorErr := meta.Accessor(obj); accessorErr == nil {
+			// Dedup owner references before updating managed fields
+			refs := accessor.GetOwnerReferences()
+			deduped, lastDuplicate := dedupOwnerReferences(refs)
+			if lastDuplicate != nil {
+				warning.AddWarning(req.Context(), "", fmt.Sprintf(DuplicateOwnerReferencesWarningFormat, lastDuplicate.UID))
```

Comment:
it seems like it would be nicer to list as many of the duplicates as you could.

Reply:
The reason was we have some [limits](https://github.com/kubernetes/kubernetes/blob/1df1f882c4421d313abb468fef5c15f89983471c/staging/src/k8s.io/apiserver/pkg/endpoints/filters/warning.go#L39-L42) that could truncate the warning message. I will change to list as many of the duplicates as we can

Reply:
done